commands/VariableCommand.js

```diff
@@ -292,12 +303,16 @@ VariableCommand.prototype = extend(BaseCommand.prototype, {
 			function checkVariable() {
 				self._getValue(result.deviceIds, result.variableName).ensure(function () {
 					setTimeout(checkVariable, delay);
-				});
+				}).catch(function() {});
 			}
 			checkVariable();
 		}).catch(function (err) {
 			self.error(err);
 		});
+	},
+
+	maximumVariableNameLength: function (deviceId) {
```

Comment:
Yeah, 1st implementation had maximum length as a function of deviceId but went back to a simple constant.